ksqldb-rest-model/pom.xml

```diff
@@ -50,6 +50,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+            <version>2.13.2.2</version>
```

Comment:
I considered using `${jackson.version}` here which we use in a few other poms like the `ksqldb-api-client` but those are for the group `com.fasterxml.jackson.datatype` so I wasn't sure if we wanted these two groups to always have to have the same version, although its possible that implicitly this happened before I added the version field

Reply:
can we introduce a property in the parent pom for this version? (e.g. `jackson.databind.version`)

Reply:
@agavra is that the proper way to do it instead of using the variable from `common`? I can do that

Reply:
defining a new variable gives us the flexibility to have different versions from `jackson.version`. Idk if that's necessary or not

Reply:
Cool, thanks! Just updated swapped to a variable in parent pom